repm/types.go

```diff
@@ -47,11 +47,6 @@ type scanItem struct {
 	Value jsnoms.Value `json:"value"`
 }
 
-type scanResponse struct {
```

Comment:
It's nice for documentation purposes. Can we do this the other way - adapt this to be the thing scan actually returns and then use it?